### PR TITLE
BAVL-439 removal of all mirgation specfic code now that the migration of old bookings to new has been done in production.

### PR DESCRIPTION
**Pull request by matt-crowson-moj**
Goodbye migration code.

Now that we have migrated the old BVLS service data into the re-platformed service we no longer need the migration specific code.

### CODE REVIEW

File: utility/display-maintenance-screen.sh
Comment (matt-crowson-moj) [inline utility/display-maintenance-screen.sh:1]: @timharrison-moj FYI

File: utility/remove-maintenance-screen.sh
Comment (matt-crowson-moj) [inline utility/remove-maintenance-screen.sh:1]: @timharrison-moj FYI

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/nomismapping/NomisMappingClient.kt [hunk 1]
```diff
@@ -13,16 +13,6 @@ class NomisMappingClient(private val nomisMappingApiWebClient: WebClient) {
     private val log = LoggerFactory.getLogger(this::class.java)
   }
 
-  @Deprecated(message = "Can be removed when migration is completed")
```
Comment (matt-crowson-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/nomismapping/NomisMappingClient.kt:16]: @timharrison-moj we do not need this, this is for migration only.

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**